homeassistant/components/upnp/device.py

```diff
@@ -67,7 +65,7 @@ async def async_create_device(
         # Register SSDP callback for updates.
         usn = f"{upnp_device.udn}::{upnp_device.device_type}"
         await ssdp.async_register_callback(
-            hass, device.async_ssdp_callback, {ssdp.ATTR_SSDP_USN: usn}
+            hass, device.async_ssdp_callback, {"usn": usn}
```

Comment:
In a future PR, should we add a constant for the callbacks?   Sonos could benefit as well.

```
sonos % git grep -A 3 register_callback 
__init__.py:            await ssdp.async_register_callback(
__init__.py-                self.hass, self._async_ssdp_discovered_player, {"st": UPNP_ST}
__init__.py-            )
```